src/main/java/com/google/songgame/servlets/RoundServlet.java

```diff
@@ -18,21 +31,135 @@
   private static final String CLIENT_KEY = "d15fbbe1c77552dc5097";
   private static final String CLIENT_SECRET = "91fd789bf568ec43d2ee";
   private static final String PUSHER_APPLICATION_NAME = "song-guessing-game";
-  private static final String PUSHER_CHAT_CHANNEL_NAME = "start-game";
+  private static final String PUSHER_ROUND_CHANNEL_NAME = "start-round";
+  private static final String PUSHER_GAME_CHANNEL_NAME = "start-game";
+  private static final int TIME_OFFSET = 3000;
+  private static final int ROUND_LENGTH = 30000;
   private Pusher pusher;
+  private Gson gson;
+  DatastoreService datastore;
 
   @Override
   public void init() {
     pusher = new Pusher(APP_ID, CLIENT_KEY, CLIENT_SECRET);
     pusher.setCluster("us2");
     pusher.setEncrypted(true);
+
+    gson = new Gson();
+    datastore = DatastoreServiceFactory.getDatastoreService();
   }
 
   @Override
-  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
```

Comment:
What is the difference between the POST and PUT methods? Does the POST start off the first round while PUT starts each next round? If that's the case, the problem is that from an API standpoint, it isn't obvious why either behavior would occur for PUT and POST. 

First off, we should consider if we even need the two separate methods. It looks like you're using PUSHER_GAME_CHANNEL_NAME to redirect to the game page, but couldn't you do that on PUSHER_ROUND_CHANNEL_NAME as well? In other words, we would only have a single pusher channel for when a new round starts (even if it's the first round), and lobby.js could use that message to redirect to the game page. In addition, game.js would use that same message to determine when to retrieve the new round info.